Fixes to ContainerGroups ARM template example

stuartpreston opened the pull request:
Updated to include valid schema reference, type name and requirement to provide resources.
Updated cli output to match resources
Minor formatting changes.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Fixes to ContainerGroups ARM template example

Updated to include valid schema reference, type name and requirement to provide resources.
Updated cli output to match resources
Minor formatting changes.
